commit 8da8197608a9f1e993f8ab217c01026d6619f4b8 (refs/changes/74/3974/2)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-05-18T16:24:00+00:00 (1 year, 5 months ago)
    
    OS-6963 add_fbuf should also use is_env_string (fix debug)
    Reviewed by: Jason King <jason.king@joyent.com>
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/usr/src/lib/brand/bhyve/zone/boot.c b/usr/src/lib/brand/bhyve/zone/boot.c
index c6431e61f6..e29616a748 100644
--- a/usr/src/lib/brand/bhyve/zone/boot.c
+++ b/usr/src/lib/brand/bhyve/zone/boot.c
@@ -479,7 +479,6 @@ add_bhyve_extra_opts(int *argc, char **argv)
 static int
 add_fbuf(int *argc, char **argv)
 {
-	char *val;
 	char conf[MAXPATHLEN];
 	int len;
 
